FlightReservation/src/consts/style.js

```diff
@@ -108,6 +112,43 @@ const style = StyleSheet.create({
     color: colors.white,
     alignSelf: 'center',
   },
+
+  //* Spinner Modal *//
+  center_item: {
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  modal_container: {
+    backgroundColor: 'rgba(0,0,0,0.5)',
```

Comment:
You can also use your Colors component here that you have already implemented.

Reply:
This way is cleaner because only affects the opacity of that specific element.